Subject: staging/hv: fill scsi/proc_name for the sake of mkinitrd
From: <ohering@suse.de>
Patch-mainline: never
References: bnc#716708

mkinitrd relies on /sys/class/scsi_host/*/proc_name instead of
/sys/block/sd*/device/../../../moalias to get the scsi driver module
name.
As a fallback the sysfs driver name could be used, which does not match
the module name either ('storvsc' vs. 'hv_storvsc').
---
 drivers/scsi/storvsc_drv.c |    1 +
 1 file changed, 1 insertion(+)

Index: linux-3.0-SLE11-SP2/drivers/scsi/storvsc_drv.c
===================================================================
--- linux-3.0-SLE11-SP2.orig/drivers/scsi/storvsc_drv.c
+++ linux-3.0-SLE11-SP2/drivers/scsi/storvsc_drv.c
@@ -1370,6 +1370,7 @@ queue_error:
 static struct scsi_host_template scsi_driver = {
 	.module	=		THIS_MODULE,
 	.name =			"storvsc_host_t",
+	.proc_name =		"hv_storvsc",
 	.bios_param =		storvsc_get_chs,
 	.queuecommand =		storvsc_queuecommand,
 	.eh_host_reset_handler =	storvsc_host_reset_handler,
